Performance/Harness.h

```diff
@@ -112,42 +118,62 @@ void Harness::measure(const Function& func) {
 
   vector<steady_clock::duration> timings;
   subtask_timings.clear();
+  bool displayed_titles = false;
+
+  printf("Running each check %d times, times in µs\n", run_count);
 
   // Do each measurement multiple times and collect the means and standard
   // deviation to account for noise.
   for (int attempt = 1; attempt <= measurement_count; attempt++) {
-    printf("Attempt %d, %d runs\n", attempt, run_count);
     current_subtasks.clear();
 
     auto start = steady_clock::now();
-    func();
+    for (auto i = 0; i < run_count; i++) {
+        subtask_names.clear();
```

Comment:
Though it may be negligible, it's worth noting that this will add the vector's `clear()` overhead into the timings we collect. (Likewise for the Swift one.)